Files/UserControls/NavigationToolbar.xaml

```diff
@@ -864,7 +864,11 @@
                 FontSize="16"
                 IsEnabled="{x:Bind IsPageTypeNotHome, Mode=OneWay}"
                 Style="{StaticResource ToolBarButtonStyle}"
-                ToolTipService.ToolTip="Search" />
+                ToolTipService.ToolTip="Search (Ctrl+F)">
+                <Button.KeyboardAccelerators>
+                    <KeyboardAccelerator Key="F" Modifiers="Control" />
```

Comment:
Can you move the keyboard accelerator to ModernShellPage to be consistent with other shortcuts?

Reply:
@yaichenbaum It doesn't really matter here because this is registered in XAML. In other words, it makes sense for that control to have its own shortcut.

Reply:
@duke7553 Unfortunately this causes issues with the shortcut triggering some of the time, this was one of the reasons we moved a lot of the shortcuts to ModernShellPage.